README.md

```diff
@@ -25,7 +25,7 @@ Then, you can depend on the extras package:
 #### Usage
 
 The OpenTelemetry Semantic Conventions package provides span attributes to work with [OpenTelemetry's
-semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions).
+semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/trace/).
```

Comment:
```suggestion
semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.11.0/specification/trace/semantic_conventions).
```

Both the old and your suggested link link to the latest version of semantic conventions. Currently, `OpenTelemetrySemanticConventions` wraps the attributes defined in v1.11.0 specifically, so how about we link to the tagged conventions as well?